modules/sdk-lib-mpc/src/tss/ecdsa/types.ts

```diff
@@ -37,6 +43,9 @@ export type DeserializedNtildeWithProofs = DeserializedNtilde & {
 export type SerializedNtildeWithProofs = SerializedNtilde & {
   ntildeProof: SerializedNtildeProofs;
 };
+export type SerializedNtildeWithVerifiers = SerializedNtilde & {
+  verifiers: NtildeVerifiers;
+};
```

Comment:
This type shouldn't exist in sdk-lib-mpc. 

`sdk-lib-mpc` is agnostic of any bitgo specific workflows/nomenclature. The signing/resigning of challenges is not an mpc primitive, but instead a bitgo abstraction. This should live in sdk-core.